modules/util/JSUtil.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Gets object's values.
+ * @param {Object} object
+ * @return {Array<Object>}
+ */
+export function getValues(object) {
```

Comment:
No support in IE, Opera, Safari. Does any of our libs fix that ? https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Object/values#Browser_compatibility